infra/ansible/roles/build_srcs/tasks/main.yaml

```diff
@@ -137,6 +137,46 @@
   ansible.builtin.pip:
     name: "{{ torchvision_wheels.files | map(attribute='path') }}"
 
+- name: Read Torchaudio pin
```

Comment:
This will make torchaudio/text whl in all build artifacts (e.g. release) as main.yaml is shared for different build. cc @will-cromar for if there is a better way to make these only apply for the test container.

Reply:
Disscussed with @will-cromar offline, he suggests we move the installation into run_tests.sh to prevent adding extra dependencies in our docker build.

Reply:
Relocated all the torchbench dependecies installation from .yml into the testing script.